package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "marigold",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "A set of common and reusable components and tools for building websites and applications at 18F.",
   "scripts": {
-    "build": "fractal build",
-    "start": "fractal start --sync"
+    "prestart": "npm run clean && npm run build",
+    "postinstall": "source env.sh",
```

Comment:
@msecret this is needed to check whether or not there is a directory for the `uswds` assets. If not, it makes one. I thought about where exactly this "check" should occur and what benefit it would have by running as a shell script too. The idea here is that the `env.sh` helps setup your build environment. In future iterations, I can see this script turning into something even more robust that helps automate more fine grained organizational components of the file/folder structure.

You would also probably want this to run at `prepublish` to so that everything builds accurately.